Sources/Mercury/Slider/MercurySlider.swift

```diff
@@ -0,0 +1,114 @@
+    
+    public init(primaryColor: Color, secondaryColor: Color,
+                primaryFont: MercuryFont, secondaryFont: MercuryFont) {
+        self.primaryColor = primaryColor
+        self.secondaryColor = secondaryColor
+        self.primaryFont = primaryFont
+        self.secondaryFont = secondaryFont
+    }
+}
+
+public struct MercuryFont {
+    var name: String
+    var size: CGFloat
+    
+    public init(name: String, size: CGFloat) {
+        self.name = name
+        self.size = size
+    }
+}
+
+public struct MercurySlider: View {
+    @Binding var progressValue: Float
+    var text: String
+    var mercurySliderDraggingEnded: (() -> Void)
+    var config: MercuryConfig
+    
+    public init(progressValue: Binding<Float>, text: String, config: MercuryConfig,
+                mercurySliderDraggingEnded: @escaping (() -> Void)) {
+        self._progressValue = progressValue
+        self.text = text
+        self.mercurySliderDraggingEnded = mercurySliderDraggingEnded
+        self.config = config
+    }
+    
+    public var body: some View {
+        VStack {
+            Text(text)
+                .font(.custom(config.primaryFont.name, size: config.primaryFont.size))
+                .foregroundColor(config.primaryColor)
+            HStack {
+                Text("Low")
+                    .font(.custom(config.secondaryFont.name, size: config.secondaryFont.size))
+                    .foregroundColor(config.secondaryColor)
+                    .padding(.leading, 16)
+                Spacer()
+                Text("High")
+                    .font(.custom(config.secondaryFont.name, size: config.secondaryFont.size))
+                    .foregroundColor(config.secondaryColor)
+                    .padding(.trailing, 16)
+            }
+            Image("TopPrgressviewbg")
+            MercuryProgressBar(progress: $progressValue, draggingEnded: {
+                mercurySliderDraggingEnded()
+            })
+            Image("BottomProgressbg")
+            Spacer()
+        }.padding(.top, 16)
+    }
+}
+
+struct MercuryProgressBar: View {
+    @Binding var progress: Float
+    var draggingEnded: (() -> Void)
+    
+    var body: some View {
+        ZStack(alignment: .leading) {
+            // Track
+            RoundedRectangle(cornerRadius: 5)
+                .foregroundColor(MercuryColor.rectangle1.swiftUIColor)
+                .frame(height: 10)
+            
+            // Progress
+            RoundedRectangle(cornerRadius: 5)
+                .foregroundColor(MercuryColor.rectangle2.swiftUIColor)
+                .frame(width: (UIScreen.main.bounds.width - 32) * CGFloat(progress), height: 10)
+            
+            // Thumb Image
+            Image("progressthumb")
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 20, height: 20)
+                .padding(.top, 4)
+                .offset(x: (UIScreen.main.bounds.width - 32) * CGFloat(progress) - 7)
```

Comment:
what is this 32?

Reply:
No idea🤧. I just copy -> paste from the main code.

Reply:
16 * 2 ?